gulp/tasks/content.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path');
+var gulp = require('gulp');
+var clone = require('gulp-clone');
+var buffer = require('vinyl-buffer');
+var mergeStream = require('merge-stream');
+var generateContentAndTemplates = require('../util/content').generateContentAndTemplates;
+var through = require('through2');
+
+gulp.task('content', function() {
+  var base, contentJSON, languagesJSON, matchHTML, templates, templateStreams;
+
+  base = path.resolve(__dirname, '..', '..');
+  contentJSON = require('../../source/content.js');
+  languagesJSON = require('../../source/languages.js');
+  matchHTML = /\.html$/;
+  templates = {};
+  templateStreams = mergeStream();
```

Comment:
You're welcome to sneak in commits to this PR if you'd like to switch the `var` structure :)

Personally I like to see a list of everything which in scope of a function right at the top, with cleaner definition lines to follow, but I know this is a preference thing.

Maybe we should put together a little JS style guide?